modules/fedimint-ln-tests/tests/tests.rs

```diff
@@ -937,6 +938,7 @@ mod fedimint_migration_tests {
                                 num_gateways > 0,
                                 "validate_migrations was not able to read any LightningGateways"
                             );
+                            info!("Validated LightningGateways");
```

Comment:
Wrong commit (ln test changes in wallet db migration commit)

Reply:
Fixed this and did my best to cleanup the rest of the commit history